### DISCUSSION TITLE
Rasterizing plots: documentation

### DISCUSSION THREAD

**Discussion by awage** [JuliaPlots/Makie.jl#1910] [Ideas]
Hello, I have seen your new feature with rasterization of plots for svg. This is a super cool feature that allows me to produce graphs like the following one. Many thanks for your effort. It would be helpful to have this keyword documented in the API. If you need a hand with it I can open a PR.

![imagen](https://user-images.githubusercontent.com/3494384/167805881-b5f2b7fb-6bb4-40e0-8854-a315dca92e13.png)

**Comment by asinghvi17** [comment]
Glad it's working out for you!  I put a paragraph in the CairoMakie section of the docs but I understand that's buried a bit deep.  Where would you recommend documenting it?  A PR would also be great :)

**Reply by awage** [reply]
I have used a `scatter` plot, but I understand that this is a common keyword to all plot types. I searched for the keyword in the API reference and it does not appear. I think it should be mentioned as a plot attribute.
